plot_app/configured_plots.py

```diff
@@ -156,6 +156,11 @@ def generate_plots(ulog, px4_ulog, db_data, vehicle_data, link_to_3d_page):
     x_range_offset = (ulog.last_timestamp - ulog.start_timestamp) * 0.05
     x_range = Range1d(ulog.start_timestamp - x_range_offset, ulog.last_timestamp + x_range_offset)
 
+    # Position focus
+    lat_focus = (data[0].data.get('lat').tolist())[0]
```

Comment:
- Can you move this block up to https://github.com/PX4/flight_review/pull/147/files#diff-5a69bf72e09d5afd2e1fcab98fff916aR143
- Can you use the same code as in https://github.com/PX4/flight_review/blob/master/plot_app/plotting.py#L310? That should be more robust.

Reply:
no problem

Reply:
@bkueng could you figure out where could I put this code block?

Reply:
Below this: https://github.com/PX4/flight_review/blob/8439f325e61ef201e1d5c620537477fb2f380a45/plot_app/configured_plots.py#L143
It should be at the center of the trajectory, and if you use `fitBounds`, you can just pass the whole trajectory.